src/components/common/panels/panels.tsx

```diff
@@ -0,0 +1,39 @@
+import classNames from 'classnames';
+import * as React from 'react';
+import * as styles from './panels.module.scss';
+
+type Panel = {
+  head: string;
+  body: string;
+};
+type Props = {
+  panels: Panel[];
+  columns?: number;
+  bodySize?: 'small' | 'regular';
+};
+
+const Panels: React.FC<Props> = ({
+  panels,
+  columns = 2,
+  bodySize = 'regular',
```

Comment:
do we really need bodySize? looks like we can get rid of it

Reply:
Yes we do. Same as columns